tomls/omnibus-base-mainnet-andromeda/collaterals/susdc.toml

```diff
@@ -5,7 +5,7 @@ defaultValue = "<%= MaxUint256 %>"
 defaultValue = "<%= parseEther('1.005') %>"
```

Comment:
@sunnyvempati if im looking at the right variable, SIP-348 has USDC liqudiation at 110% https://sips.synthetix.io/sips/sip-348/

Reply:
yeah it shouldn't be this i will update the sip today, with borrow disabled it doesn't need to be high